### ISSUE TITLE
DjangoListField error in 0.4.7

### ISSUE THREAD

**Issue by leot78** [eamigo86/graphene-django-extras#124]
Hi,

I have updated graphene-django to 2.6.0 and updated graphene-django-extras to 0.4.7 and now i am having this error : 
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.7/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/django/views/generic/base.py", line 62, in view
    self = cls(**initkwargs)
  File "/usr/local/lib/python3.7/site-packages/graphene_django/views.py", line 79, in __init__
    schema = graphene_settings.SCHEMA
  File "/usr/local/lib/python3.7/site-packages/graphene_django/settings.py", line 117, in __getattr__
    val = perform_import(val, attr)
  File "/usr/local/lib/python3.7/site-packages/graphene_django/settings.py", line 56, in perform_import
    return import_from_string(val, setting_name)
  File "/usr/local/lib/python3.7/site-packages/graphene_django/settings.py", line 70, in import_from_string
    module = importlib.import_module(module_path)
  File "/usr/local/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/src/app/unknow_api/schema.py", line 2, in <module>
    from api import queries, mutations
  File "/usr/src/app/api/queries.py", line 43, in <module>
    class CommentListType(DjangoListObjectType):
  File "/usr/local/lib/python3.7/site-packages/graphene/utils/subclass_with_meta.py", line 52, in __init_subclass__
    super_class.__init_subclass_with_meta__(**options)
  File "/usr/local/lib/python3.7/site-packages/graphene_django_extras/types.py", line 297, in __init_subclass_with_meta__
    result_container = DjangoListField(baseType)
  File "/usr/local/lib/python3.7/site-packages/graphene_django/fields.py", line 24, in __init__
    ), "DjangoListField only accepts DjangoObjectType types"
AssertionError: DjangoListField only accepts DjangoObjectType types
```
I've seen that in 0.4.7 there is this commit [0e7a6f86e2c225b](https://github.com/eamigo86/graphene-django-extras/commit/0e7a6f86e2c225bb97d45439e8a5707e0c38089a) that seems to fix the problem so i don't why i'm still having this error.

Thank you for this amazing library !

**Comment by leot78** [comment]
Shouldn't this [line](https://github.com/eamigo86/graphene-django-extras/blob/c9eb5a896b467b83f3f708820188890f109962b6/graphene_django_extras/types.py#L12)  be replaced  by `from graphene_django_extras.fields import DjangoListField` ?

**Comment by kokokh** [comment]
I am receiving the same error too, for now I have reverted back to graphene-django 2.5.0.

**Comment by eamigo86** [comment]
Hi @kokokh, has been this error with version 0.4.8?

**Comment by kokokh** [comment]
> Hi @kokokh, has been this error with version 0.4.8?

yes, I updated from 0.4.6. it seems to work without any issues with graphene-django==2.5.0, but not 2.6.0

**Comment by axwalker** [comment]
I'm also getting this error with:

graphene-django==2.8.0
graphene-django-extras = "0.4.8"

whenever I try to use `DjangoSerializerType`.

**Comment by Faisal-Manzer** [comment]
Same error. Is anyone have any workaround? @eamigo86 
Is this fixed in master?

**Comment by shockflash** [comment]
Workaround: Use pagination

`class MyType(DjangoSerializerType):
    # Main class that brings everything together and defines the filter for the query

    class Meta:
        serializer_class = MySerializer
        model = MyModel
        pagination = LimitOffsetGraphqlPagination(default_limit=25) # ordering can be: string, tuple or list`

**Comment by eamigo86** [comment]
This error should be corrected in the recently released version 0.4.9.
